labgrid/driver/usbsdwiredriver.py

```diff
@@ -37,3 +38,15 @@ def set_mode(self, mode):
             self.mux.control_serial,
         ]
         processwrapper.check_output(cmd)
+
+    @Driver.check_active
+    @step(title='sdmux_get')
+    def get_mode(self):
+        cmd = self.mux.command_prefix + [
+            self.tool,
+            "-e",
+            self.mux.control_serial,
+            "-u",
+        ]
+        result = processwrapper.check_output(cmd)
+        return re.sub('^.*?:\s*', "", result.strip().decode())
```

Comment:
Looking at the [code](https://git.tizen.org/cgit/tools/testlab/sd-mux/tree/src/main.cpp?id=9dd189d973da64e033a0c5c2adb3d94b23153d94#n583), I guess this can be simplified:

```suggestion
        return result.split(": ", maxsplit=1)[1].strip().decode()
```
The `re` import can be dropped.

Reply:
Sure.  Do you not have the option to squash when you merge the pull request?

Reply:
Thanks. We haven't enabled that option. I don't like dropping logical commit separation accidentally and I don't think it's necessary to fiddle with your commits if I don't have to :)